[IMP] inventory: update landed costs doc (#8535)
odoo/documentation: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_valuation/integrating_landed_costs.rst

============
Landed costs
============

.. _inventory/reporting/landed_costs:

.. |RfQ| replace:: :abbr:`RfQ (Request for Quotation)`
.. |PO| replace:: :abbr:`PO (Purchase Order)`
.. |FIFO| replace:: :abbr:`FIFO (First In First Out)`
.. |AVCO| replace:: :abbr:`AVCO (Average Costing)`

When shipping products to customers, the landed cost is the total price of a product or shipment,
including all expenses associated with shipping the product.

> In Odoo, the *Landed Costs* feature allows users to include additional costs into the cost of the
> product. This includes the cost of shipment, insurance, customs duties, taxes, and other fees.

Reviewer: ```suggestion
In Odoo, the *Landed Costs* feature is used to take into account additional costs when calculating the valuation of a product. This includes the cost of shipment, insurance, customs duties, taxes, and other fees.
```

Author: Suggestion implemented ✅

To do this, create a new product by going to :menuselection:`Inventory app --> Products -->
Products`, and clicking :guilabel:`New`.

Assign a name to the landed cost product in the :guilabel:`Product Name` field (i.e.
international shipping). In the :guilabel:`Product Type` field, click the drop-down menu and
select :guilabel:`Service`. Landed cost products **must** be set as service type products.

Click the :guilabel:`Purchase` tab, and tick the checkbox next to :guilabel:`Is a Landed Cost`. Once
ticked, a new :guilabel:`Default Split Method` field appears below it, prompting a selection.
Clicking the drop-down menu reveals the following options:

- :guilabel:`Equal`: splits the cost equally across each product included in the receipt, regardless
  of the quantity of each.
- :guilabel:`By Quantity`: splits the cost across each unit of all products in the receipt.
- :guilabel:`By Current Cost`: splits the cost according to the cost of each product unit, so a
  product with a *higher* cost receives a *greater* share of the landed cost.
- :guilabel:`By Weight`: splits the cost according to the weight of the products in the receipt.

> - :guilabel:`By Volume`: splits the cost according to the volume of the products included in the

Reviewer: ```suggestion
- :guilabel:`By Volume`: splits the cost according to the volume of the products in the
```

Author: Suggestion implemented ✅

  receipt.

.. image:: integrating_landed_costs/integrating-landed-costs-landed-cost-product.png
   :align: center
   :alt: Is a Landed Cost checkbox and Default Split Method on service type product form.

Once ready, click :guilabel:`Save` to save the new product. When creating new vendor bills, this
product can be added as an invoice line as a landed cost.

.. important::
   To apply a landed cost on a vendor bill, the products included in the original |PO| **must**
   belong to a *product category* with its *Force Removal Strategy* set to |FIFO|, and its *Costing
   Method* set to |AVCO|.

Create purchase order
=====================

Navigate to :menuselection:`Purchase app --> New` to create a new request for quotation (RfQ). In

> the :guilabel:`Vendor` field, add a vendor to order products form. Then, click :guilabel:`Add a

Reviewer: ```suggestion
the :guilabel:`Vendor` field, add a vendor to order products from. Then, click :guilabel:`Add a
```

Author: Suggestion implemented ✅

product` under the :guilabel:`Products` tab to add products to the |RfQ|.

Once ready, click :guilabel:`Confirm Order` to confirm the order. Then, click :guilabel:`Receive
Products`, followed by :guilabel:`Validate`.

Create vendor bill
------------------

Once the vendor fulfills the |PO| and sends a bill, a vendor bill can be created from the |PO| in
Odoo.

Navigate to the :menuselection:`Purchase app`, and click into the |PO| for which a vendor bill
should be created. Then, click :guilabel:`Create Bill`. This opens a new :guilabel:`Draft Bill`
form.

In the :guilabel:`Bill Date` field, click the line to open a calendar popover menu, and select the

> date for which this draft bill should be billed.

Reviewer: ```suggestion
date on which this draft bill should be billed.
```

Author: Suggestion implemented ✅

Then, under the :guilabel:`Invoice Lines` tab, click :guilabel:`Add a line`, and click the drop-down

> menu to select the previously-created landed cost product. Click :guilabel:`Save` to update the

Reviewer: ```suggestion
menu to select the previously created landed cost product. Click :guilabel:`Save` to update the
```

Author: Suggestion implemented ✅

.. image:: integrating_landed_costs/integrating-landed-costs-transfers-menu.png
   :align: center
   :alt: Landed cost form with selected receipt transfer.

.. tip::
   In addition to creating landed costs directly from a vendor bill, landed cost records can *also*
   be created by navigating to :menuselection:`Inventory app --> Operations --> Landed Costs`, and
   clicking :guilabel:`New`.

After setting the picking on the landed cost record, click :guilabel:`Compute` (at the bottom of the
form, under the :guilabel:`Total:` cost).

Click the :guilabel:`Valuation Adjustments` tab to see the impact of the landed costs. The
:guilabel:`Original Value` column lists the original price of the |PO|, the :guilabel:`Additional
Landed Cost` column displays the landed cost, and the :guilabel:`New Value` displays the sum of the
two, for the total cost of the |PO|.

Once ready, click :guilabel:`Validate` to post the landed cost entry to the accounting journal.

This causes a :guilabel:`Valuation` smart button to appear at the top of the form. Click the
:guilabel:`Valuation` smart button to open a :guilabel:`Stock Valuation` page, with the product's

> updated valuation listed.

Reviewer: It might be good to add a note mentioning that you can view the valuation of every product, including landed costs, by navigating to `Inventory --> Reporting --> Valuation`.

Author: Suggestion implemented ✅

Tick the checkbox next to the :guilabel:`Landed Costs` option, and click :guilabel:`Save` to save
changes.

Once the page refreshes, a new :guilabel:`Default Journal` field appears below the :guilabel:`Landed
Costs` feature in the :guilabel:`Valuation` section.

Click the :guilabel:`Default Journal` drop-down menu to reveal a list of accounting journals. Select
a journal for which all accounting entries related to landed costs should be recorded.

.. image:: integrating_landed_costs/integrating-landed-costs-enabled-setting.png
   :align: center
   :alt: Landed Costs feature and resulting Default Journal field in the Inventory settings.

Create landed cost product
==========================

For charges that are consistently added as landed costs, a landed cost product can be created in
Odoo. This way, a landed cost product can be quickly added to a vendor bill as an invoice line,
instead of having to be manually entered every time a new vendor bill is created.

To do this, create a new product by going to :menuselection:`Inventory app --> Products -->
Products`, and clicking :guilabel:`New`.

Assign a name to the landed cost product in the :guilabel:`Product Name` field (i.e.

> International Shipping). In the :guilabel:`Product Type` field, click the drop-down menu, and select

Reviewer 2: ```suggestion
`International Shipping`). In the :guilabel:`Product Type` field, click the drop-down menu, and select
```
using single ticks here because I'm under the impression that's the formatting we use when suggesting that something be "entered" as a name/entry in a form field.

Author: Suggestion implemented ✅

> :guilabel:`Service` as the :guilabel:`Product Type`. Landed cost products **must** be set as service

Reviewer 2: the mention of it having to be set as service product types might be worthy of an 'important' admonition block

I'd also consider rephrasing it so it uses appropriate GUIs/terms:
"Landed cost products **must** have their :guilabel:`Product Type` set on :guilabel:`Service`."

Author: Nice catch - Suggestion implemented ✅

============
Landed costs
============

.. _inventory/reporting/landed_costs:

.. |RfQ| replace:: :abbr:`RfQ (Request for Quotation)`
.. |PO| replace:: :abbr:`PO (Purchase Order)`
.. |FIFO| replace:: :abbr:`FIFO (First In First Out)`
.. |AVCO| replace:: :abbr:`AVCO (Average Costing)`

When shipping products to customers, the landed cost is the total price of a product or shipment,
including all expenses associated with shipping the product.

In Odoo, the *Landed Costs* feature is used to take additional costs into account when calculating
the valuation of a product. This includes the cost of shipment, insurance, customs duties, taxes,
and other fees.

Configuration
=============

To add landed costs to products, the *Landed Costs* feature must first be enabled. To enable this
feature, navigate to :menuselection:`Inventory app --> Configuration --> Settings`, and scroll to
the :guilabel:`Valuation` section.

> Click the checkbox next to the :guilabel:`Landed Costs` option, and click :guilabel:`Save` to save

Author: Suggestion implemented ✅

changes.

Once the page refreshes, a new :guilabel:`Default Journal` field appears below the :guilabel:`Landed

> Costs` feature's checkbox in the :guilabel:`Valuation` section.

Author: Suggestion implemented ✅

> Click the drop-down menu to reveal a list of accounting journals. Select a journal for which all

Author: Suggestion implemented ✅

accounting entries related to landed costs should be recorded.

.. image:: integrating_landed_costs/integrating-landed-costs-enabled-setting.png
   :align: center
   :alt: Landed Costs feature and resulting Default Journal field in the Inventory settings.

Create landed cost product
==========================

For charges that are consistently added as landed costs, a landed cost product can be created in
Odoo. This way, a landed cost product can be quickly added to a vendor bill as an invoice line,
instead of having to be manually entered every time a new vendor bill is created.

To do this, create a new product by going to :menuselection:`Inventory app --> Products -->
Products`, and clicking :guilabel:`New`.

Assign a name to the landed cost product in the :guilabel:`Product Name` field (i.e.

> international shipping). In the :guilabel:`Product Type` field, click the drop-down menu and

Author: Suggestion implemented ✅

> select :guilabel:`Service`. Landed cost products **must** be set as service type products.

Author: Suggestion implemented ✅

> Click the :guilabel:`Purchase` tab, and tick the checkbox next to :guilabel:`Is a Landed Cost`. Once

Author: Suggestion implemented ✅

ticked, a new :guilabel:`Default Split Method` field appears below it, prompting a selection.

> Clicking the drop-down menu reveals the following options:

Author: Suggestion implemented ✅

- :guilabel:`Equal`: splits the cost equally across each product included in the receipt, regardless
  of the quantity of each.
- :guilabel:`By Quantity`: splits the cost across each unit of all products in the receipt.
- :guilabel:`By Current Cost`: splits the cost according to the cost of each product unit, so a

>   product with a *higher* cost receives a *greater* share of the landed cost.

Author: Suggestion implemented ✅

- :guilabel:`By Weight`: splits the cost according to the weight of the products in the receipt.

> - :guilabel:`By Volume`: splits the cost according to the volume of the products in the receipt.

Author: Suggestion implemented ✅

.. image:: integrating_landed_costs/integrating-landed-costs-landed-cost-product.png
   :align: center
   :alt: Is a Landed Cost checkbox and Default Split Method on service type product form.

> Once ready, click :guilabel:`Save` to save the new product. When creating new vendor bills, this

Author: Correct - removing that part. Suggestion implemented ✅

product can be added as an invoice line as a landed cost.

.. important::
   To apply a landed cost on a vendor bill, the products included in the original |PO| **must**

>    belong to a *product category* with its *Force Removal Strategy* set to |FIFO|, and its *Costing

Author: Suggestion implemented ✅

   Method* set to |AVCO|.

Create purchase order
=====================

Navigate to :menuselection:`Purchase app --> New` to create a new request for quotation (RfQ). In
the :guilabel:`Vendor` field, add a vendor to order products from. Then, click :guilabel:`Add a

> product` under the :guilabel:`Products` tab to add products to the |RfQ|.

Author: Suggestion implemented ✅

Once ready, click :guilabel:`Confirm Order` to confirm the order. Then, click :guilabel:`Receive

> Products`, followed by :guilabel:`Validate`.

Author: Suggestion implemented ✅

Create vendor bill
------------------

Once the vendor fulfills the |PO| and sends a bill, a vendor bill can be created from the |PO| in
Odoo.

Navigate to the :menuselection:`Purchase app`, and click into the |PO| for which a vendor bill
should be created. Then, click :guilabel:`Create Bill`. This opens a new :guilabel:`Draft Bill`

> form.

Author: Suggestion implemented ✅

In the :guilabel:`Bill Date` field, click the line to open a calendar popover menu, and select the
date on which this draft bill should be billed.

Then, under the :guilabel:`Invoice Lines` tab, click :guilabel:`Add a line`, and click the drop-down

> menu to select the previously created landed cost product. Click :guilabel:`Save` to update the

Author: Suggestion implemented ✅

draft bill.

.. image:: integrating_landed_costs/integrating-landed-costs-checkboxes.png
   :align: center
   :alt: Landed Costs column checkboxes for product and landed cost.

In the :guilabel:`Landed Costs` column, the product ordered from the vendor does **not** have its
checkbox ticked, while the landed cost product's checkbox **is** ticked. This differentiates landed
costs from all other costs displayed on the bill.

Additionally, at the top of the form, a :guilabel:`Create Landed Costs` button appears.

.. image:: integrating_landed_costs/integrating-landed-costs-create-button.png
   :align: center
   :alt: Create Landed Costs button on vendor bill.

Add landed cost
===============

Once a landed cost is added to the vendor bill, click :guilabel:`Create Landed Costs` at the top of
the vendor bill.

> Doing so automatically creates a landed cost record with a set landed cost pre-filled in the product

Author: Suggestion implemented ✅

line in the :guilabel:`Additional Costs` tab.

> From here, click the :guilabel:`Transfers` drop-down menu, and select which transfer the landed cost

Author: Suggestion implemented ✅

belongs to.

> Once ready, click :guilabel:`Save` to save changes.

Author: Nope, removing it. Suggestion implemented ✅

.. image:: integrating_landed_costs/integrating-landed-costs-transfers-menu.png
   :align: center
   :alt: Landed cost form with selected receipt transfer.

.. tip::
   In addition to creating landed costs directly from a vendor bill, landed cost records can *also*
   be created by navigating to :menuselection:`Inventory app --> Operations --> Landed Costs`, and
   clicking :guilabel:`New`.

> After setting the picking on the landed cost record, click :guilabel:`Compute` (at the bottom of the

Author: That's correct - added to the sentence to clarify that. Suggestion implemented ✅

form, under the :guilabel:`Total:` cost).

Click the :guilabel:`Valuation Adjustments` tab to see the impact of the landed costs. The
:guilabel:`Original Value` column lists the original price of the |PO|, the :guilabel:`Additional
Landed Cost` column displays the landed cost, and the :guilabel:`New Value` displays the sum of the
two, for the total cost of the |PO|.

Once ready, click :guilabel:`Validate` to post the landed cost entry to the accounting journal.

This causes a :guilabel:`Valuation` smart button to appear at the top of the form. Click the
:guilabel:`Valuation` smart button to open a :guilabel:`Stock Valuation` page, with the product's

> updated valuation listed.

Reviewer 2: I'm actually not seeing any 'Valuation' smart button appear after I complete this process. Is there any reason/restriction that exists as to why this isn't showing up? Some elaboration/clarification on this would be helpful. Thanks.

Author: The *only* reason I can think of as to why that would happen is if the product being used for this testing is a Consumable, and not a Storable product. All my tests yield the Valuation smart button otherwise

Reviewer 2: Understood. It may be wise to add a brief note about the possibility of this happening -- just in case a reader falls into the same issue. 👍

Author: That's a nice call - I added in a note below this paragraph to explain that. Suggestion implemented ✅

To view the valuation of *every* product, including landed costs, navigate to
:menuselection:`Inventory app --> Reporting --> Valuation`.

.. note::
   Each journal entry created for a landed cost on a vendor bill can be viewed in the *Accounting*
   app.

   To locate these journal entries, navigate to :menuselection:`Accounting app --> Accounting -->

>    Journal Entries`, and locate the correct entry, by number (i.e. **PBNK1/2024/XXXXX**).

Reviewer 2: is there a reason the same entry is bolded? Would `XXXX/XXXX/XXXX` be more appropriate? What are your thoughts?

>    Click into the journal entry to view the *Journal Items*, and other information about the entry.

Reviewer 2: ```suggestion
   Click into the journal entry to view the :guilabel:`Journal Items`, and other information about the entry.
```

Author: Suggestion implemented ✅

the :guilabel:`Valuation` section.

Tick the checkbox next to the :guilabel:`Landed Costs` option, and click :guilabel:`Save` to save
changes.

Once the page refreshes, a new :guilabel:`Default Journal` field appears below the :guilabel:`Landed
Costs` feature in the :guilabel:`Valuation` section.

Click the :guilabel:`Default Journal` drop-down menu to reveal a list of accounting journals. Select
a journal for which all accounting entries related to landed costs should be recorded.

.. image:: integrating_landed_costs/integrating-landed-costs-enabled-setting.png
   :align: center
   :alt: Landed Costs feature and resulting Default Journal field in the Inventory settings.

Create landed cost product
==========================

For charges that are consistently added as landed costs, a landed cost product can be created in
Odoo. This way, a landed cost product can be quickly added to a vendor bill as an invoice line,
instead of having to be manually entered every time a new vendor bill is created.

To do this, create a new product by going to :menuselection:`Inventory app --> Products -->
Products`, and clicking :guilabel:`New`.

> Assign a name to the landed cost product in the :guilabel:`Product Name` field (i.e.
> `International Shipping`). In the :guilabel:`Product Type` field, click the drop-down menu, and
> select :guilabel:`Service` as the :guilabel:`Product Type`.

Reviewer 3: early line break @ 49
```suggestion
Assign a name to the landed cost product in the :guilabel:`Product Name` field (i.e. `International
Shipping`). In the :guilabel:`Product Type` field, click the drop-down menu, and select
:guilabel:`Service` as the :guilabel:`Product Type`.
```
